browser/js/common/directives/kit-fields/kit-code-field.html

```diff
@@ -0,0 +1,9 @@
+<section id="kit-code-field" ng-controller="AceCtrl">
```

Comment:
The controller is embedded with `ng-controller`. I'd recommend simply attaching the controller to the directive directly, with`{...controller: 'AceCtrl'...}` over in the directive's definition. Or (slightly) better yet, throw the logic into the directive's link function. Either way, I think it's nice to have the directive definition reference all of the directive's JS logic. So that, for example, if you're trying to trace the execution flow in your directive, there's an explicit trail to follow.